src/txn/txn_rollback_to_stable.c

```diff
@@ -1156,7 +1148,9 @@ __rollback_to_stable_btree_apply(WT_SESSION_IMPL *session)
     char ts_string[2][WT_TS_INT_STRING_SIZE];
     const char *config, *uri;
     bool durable_ts_found, prepared_updates;
+    size_t addr_size;
```

Comment:
Declare the variables according to their size. Move the size_t before all the char. Change the order of the initialization also.

Reply:
Changed the order of declaration and initialization.